install.ps1

```diff
@@ -0,0 +1,56 @@
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+$ProgressPreference = 'SilentlyContinue'
+$release = Invoke-RestMethod -Method Get -Uri "https://api.github.com/repos/ducaale/xh/releases/latest"
+$asset = $release.assets | Where-Object name -like *.zip
+$destdir = "$home\bin\"
+$zipfile = "$env:TEMP\$($asset.name)"
+
+Write-Output "Downloading: $($asset.name)"
+Invoke-RestMethod -Method Get -Uri $asset.browser_download_url -OutFile $zipfile
+
+# checks if an older version of xh.exe exists in '$destdir', if yes, then delete it, if not, then download latest zip to extract from
+$xhPath = "${destdir}xh.exe"
+if (Test-Path -Path $xhPath -PathType Leaf) {
+    "`n xh.exe exists in $destdir, deleting..."
+	rm -r -fo $xhPath
+}
+
+#xh.exe extraction start
+Add-Type -Assembly System.IO.Compression.FileSystem
+
+$zip = [IO.Compression.ZipFile]::OpenRead($zipfile)
+$entries=$zip.Entries | where {$_.FullName -like '*.exe'} 
+
+#create dir for result of extraction
+New-Item -ItemType Directory -Path $destdir -Force | Out-Null
+
+#extraction
+$entries | foreach {[IO.Compression.ZipFileExtensions]::ExtractToFile( $_, $destdir + $_.Name) }
+#free zipfile
+$zip.Dispose()
+
+Remove-Item -Path $zipfile
+
+Write-Host "`n Exctracted 'xh.exe' file is located in: $destdir."
+
+# Requires powershell to be run as administrator. Creates 'xhs' symbolic link
+cmd /c mklink /d xhs $xhPath
```

Comment:
I think you meant to use `/H` instead `/d`, which creates a soft link pointing to a directory?
```
cmd /c mklink /H xhs.exe $xhPath
```

However, requiring administrator mode might be inconvenient for users, so how about copying the binary and renaming that to `xhs.exe` instead?

Reply:
If I understood correctly, you meant to copy xh.exe as xhs.exe? Created latest commit with it and it worked on a clean windows virtual machine. Image included. Also removed the press any key at the end.

It is incredibly fast now so all the Write-Host and other comments it sends to Powershell are now pointless. I did not remove them though.

![image](https://user-images.githubusercontent.com/73487474/150126993-37a66585-2912-4aed-9d5e-18bf88c7b9cd.png)